core/domain/stats_jobs_one_off.py

```diff
@@ -340,7 +350,9 @@ def reduce(exp_id, stringified_values):
                         answer_group.outcome.dest].first_hit_count_v1
                     for answer_group in init_state.interaction.answer_groups
                     if answer_group.outcome.dest != (
-                        versioned_exploration.init_state_name)] or [0])
+                        versioned_exploration.init_state_name) and (
+                            answer_group.outcome.dest in (
```

Comment:
Add comment above the whole thing explaining something along the lines of: "we check that answer_group.outcome.dest is in the list of states because some older explorations use an implicit, pseudo-END state."